enterprise/server/cmd/goinit/main.go

```diff
@@ -330,6 +345,14 @@ func main() {
 			die(err)
 		}
 	}
+	die(sysctl(map[string]string{
+		// Default value, but set it explicitly to be sure.
+		"fs.inotify.max_user_instances": "128",
```

Comment:
I think this is already 128 in our firecracker VMs, so probably ok to remove this?

Verified by running vmstart (`blaze run enterprise/tools/vmstart`) and:
```
~ # sysctl fs.inotify.max_user_instances
fs.inotify.max_user_instances = 128
~ #
```